graylog2-shared/src/main/java/org/graylog2/shared/buffers/JournallingMessageHandler.java

```diff
@@ -58,14 +62,30 @@ public void onEvent(RawMessageEvent event, long sequence, boolean endOfBatch) th
 
             final Converter converter = new Converter();
```

Comment:
This could/should be a `static final` class member to prevent unnecessary allocation on each message event.